sdk/communication/azure-communication-jobrouter/setup.py

```diff
@@ -65,6 +65,7 @@
     install_requires=[
         "isodate<1.0.0,>=0.6.1",
         "azure-core<2.0.0,>=1.28.0",
+        "msrest>=0.7.1",
```

Comment:
Please remove this.

Reply:
@annatisch Removed for all SDKs excluding SMS SDK as msrest.serialization.Model is a parent class of [SmsSendResult](https://github.com/Azure/azure-sdk-for-python/blob/814bbb769d49ab0a0b71ab06e8e57ccdb75b2e02/sdk/communication/azure-communication-sms/azure/communication/sms/_models/_models_py3.py#L14) model. I could not find similar object in azure.core, so I will leave it for the ACS SMS team to get rid of this dependency :)